nextflow.config

```diff
@@ -46,6 +46,7 @@ params {
 
     // Alignment options
     star_ignore_sjdbgtf                  = false
+    star_limit_bam_sort_ram              = 0
```

Comment:
Are we sure it is not going to be 0 if not set?

Reply:
nope `0` corresponds to `false` in Groovy so the check in the config will see this as a `false` value

Reply:
I added `> 0` to the check to make it more clear